cray_product_catalog/catalog_delete.py

```diff
@@ -155,27 +315,48 @@ def modify_config_map(name, namespace, product, product_version, key=None):
                 name, namespace, client.V1ConfigMap(data=config_map_data)
             )
             LOGGER.info("ConfigMap update attempt %s successful", attempt)
-        except ApiException:
-            LOGGER.exception("Error calling patch_namespaced_config_map")
+        except ApiException as e:
+            if e.status == ERR_CONFLICT:
```

Comment:
Linter doesn't like single letter variables.
```suggestion
        except ApiException as exc:
            if exc.status == ERR_CONFLICT:
```